src/idb_database/idb_version_change_event.rs

```diff
@@ -24,9 +26,14 @@ impl IdbVersionChangeEvent {
             .expect("Failed to unwrap IdbOpenDbRequest result")
             .unchecked_into();
 
+        let req: IdbRequest = event.target().expect("Failed to unwrap IdbOpenDbRequest event target as IdbRequest")
```

Comment:
No point going through `event.target()` again when you can just cast the existing `req`

Reply:
Patched it up, I'll be doing some other refactors before publishing the release so I'm not too worried about the formatting.